### PR TITLE
Organize storybook components

### PR DESCRIPTION
**Pull request by andrew-frazier6**
Created folders in order to better keep track of the components. I believe this to be a good start for the structuring and if you need me to go a little further and be more precise I can. Open to suggestions

### CODE REVIEW

File: client/src/components/Icon/Icon.stories.js [hunk 1]
```diff
@@ -36,7 +36,11 @@ library.add(
   faEdit
 );
 
-storiesOf("Icon", module)
+export default {
+  title: "Layout/Icon"
+};
+
+storiesOf("Layout/Icon", module)
```
Comment (ZakkMan) [inline client/src/components/Icon/Icon.stories.js]: Let's move the Icon component to Atoms

File: client/src/components/Logo/Logo.stories.js [hunk 2]
```diff
@@ -2,7 +2,11 @@ import React from "react";
 import { storiesOf } from "@storybook/react";
 import Logo from "./Logo";
 
-storiesOf("Logo", module)
+export default {
+  title: "Layout/Logo"
+};
+
+storiesOf("Layout/Logo", module)
```
Comment (ZakkMan) [inline client/src/components/Logo/Logo.stories.js]: Let's move the Logo component to Atoms

File: client/src/components/ExtraUsers/ExtraUsers.stories.js [hunk 3]
```diff
@@ -1,10 +1,14 @@
...
 
-storiesOf('ExtraUsers', module)
-  .add("4 Extra Users ", () => <ExtraUsers to='Inbox/Recipients' users={anArray}/>)
-  .add("2 Extra Users", () => <ExtraUsers to='Inbox/Recipients' users={anArray.slice(0,6)}/>)
-  .add("No Extra Users", () => <ExtraUsers to='Inbox/Recipients' users={anArray.slice(0,3)}/>)
+export default {
+  title: "Layout/ExtraUsers"
+};
+
+storiesOf("Layout/ExtraUsers", module)
```
Comment (ZakkMan) [inline client/src/components/ExtraUsers/ExtraUsers.stories.js]: Let's move ExtraUsers to Atoms

File: client/src/components/NavLink/NavLink.stories.js [hunk 4]
```diff
@@ -1,24 +1,28 @@
...
   }
-]
+];
 
-storiesOf('NavLink', module)
+export default {
+  title: "Layout/NavLink"
+};
+
+storiesOf("Layout/NavLink", module)
```
Comment (ZakkMan) [inline client/src/components/NavLink/NavLink.stories.js]: I think NavLink and NavLinkDropdownMenu should maybe go under Atoms and if possible group them together somehow under Atoms

File: client/src/components/ProfilePicture/ProfilePicture.stories.js [hunk 5]
```diff
@@ -1,13 +1,17 @@
...
+import pic3 from "../../assets/images/profile-picture-3.png";
+import pic4 from "../../assets/images/profile-picture-4.png";
+import pic5 from "../../assets/images/profile-picture-5.png";
 
-storiesOf("Profile Picture", module)
+export default {
+  title: "Layout/Profile Picture"
+};
+
+storiesOf("Layout/Profile Picture", module)
```
Comment (ZakkMan) [inline client/src/components/ProfilePicture/ProfilePicture.stories.js]: Let's put ProfilePicture under Atoms too

File: client/src/components/Card/Card.stories.js [hunk 6]
```diff
@@ -7,7 +7,11 @@ import people from "../../assets/images/add-friend.png";
 import trends from "../../assets/images/trending-hashtags.png";
 import groups from "../../assets/images/protest.png";
 
-storiesOf("Card", module)
+export default {
+  title: "Cards/Card"
+};
```
Comment (ZakkMan) [inline client/src/components/Card/Card.stories.js]: I don't think we need this. Can you go through and delete these from the components?

**Comment by andrew-frazier6** [comment]
@ZakkMan @Underwaterr 
 I believe all the changes have been made. The unwanted lines are the removal of the export default. I think you were right. As far as I checked into it seems that that is mostly used for more complicated issues, So great call on not needing that. I know now for next time

**Pull request merged**